packages/cli/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "vc-http-api-cli",
+  "version": "1.0.0",
+  "description": "CLI for vc-http-api implementation",
+  "main": "src/index.js",
+  "bin": {
+    "vc-util": "dist/index.js"
+  },
+  "scripts": {
+    "test": "echo \"This CLI for the moment relies solely on the tests in the main services\" && exit 0",
```

Comment:
sadge, no tests.

Reply:
yep, will add once we have some more functionality to actually test - likely command line parameter validation with demo data